Assets/Scripts/DDAA/Conditions/LevelProgressionCondition.cs

```diff
@@ -29,7 +29,7 @@ public static LevelProgressionCondition Instance
     }
 
     public int currentLevel;
-    private static readonly int[] expectedFinishTimes = new int[] { 150, 330, 550 }; // adjust these. Expected time (in sec), when player should complete level
+    private static readonly int[] expectedFinishTimes = new int[] { 50, 100, 180 }; // adjust these. Expected time (in sec), when player should complete level
```

Comment:
was this done for testing? Too fast

Reply:
that doesn't really matter much as it's one of the things to be adjusted when testing DDA anyway

Reply:
Those are just very arbitrary numbers right now